infra/watchdog/batcher/README.md

```diff
@@ -0,0 +1,36 @@
+# Batcher Watchdog
+
+The Batcher Watchdog checks a prometheus metric and restart the batcher as needed
+
+## Configuration
+
+You need to create a .env file with the following variables
+
+```
+PROMETHEUS_URL=<ip>:<port>
+SYSTEMD_SERVICE=batcher
+PROMETHEUS_COUNTER=sent_batches
+PROMETHEUS_BOT=batcher
+PROMETHEUS_INTERVAL=20m
+SLACK_WEBHOOK_URL=<>
+```
+
+There is a `.env.example` file in this directory.
+
+## Run with Crontab
+
+Open the Crontab configuration with `crontab -e` and add the following line:
+
+```
+*/20 * * * * /path/to/watchdog/batcher_watchdog.sh /path/to/config/.env >> /path/to/logs/folder/batcher_watchdog.log 2>&1
```

Comment:
Shouldn't this run every 10 minutes rather than 20? 20 minutes is the hard deadline, right? With 20 minutes here we could be hanged up up to 40' until we find out about it.

Reply:
Yes, you are right, it has to be defined to PROMETHEUS_INTERVAL/2